tests/README.md

```diff
@@ -79,13 +91,11 @@ For executing the PySpark integration tests:
 
 ## How to create tests on notebooks with Papermill
 
-In the notebooks of these repo we use [Papermill](https://github.com/nteract/papermill) in unit, smoke and integration tests. 
-
-In the unit tests we just make sure the notebook runs. In the smoke tests, we run them with a small dataset or a small number of epochs to make sure that, apart from running, they provide reasonable metrics. Finally, in the integration tests, we use a bigger dataset for more epochs and we test that the metrics are what we expect. 
+In the notebooks of this repo, we use [Papermill](https://github.com/nteract/papermill) in unit, smoke and integration tests. 
```

Comment:
Add:  Papermill is a tool that enables you to parameterize notebooks, execute and collect metrics across the notebooks, and summarize collections of notebooks;